From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 13 Nov 2018 09:59:30 -0800
Subject: [PATCH] Use system dmlc-core

[Update for 2019-08-01]: Rebased against 1.5.0; do not attempt to
                         install the 3rd-party headers either
[Update for 2020-06-23]: Rebased against 1.6.0
[Update for 2020-10-19]: Rebased against 1.7.0
[Update for 2020-03-08]: Rebased against 1.8.0

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ab20736..cbd25b9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -337,7 +337,6 @@ endforeach()
 include_directories("include")
 include_directories("3rdparty/tvm/nnvm/include")
 include_directories("3rdparty/tvm/include")
-include_directories("3rdparty/dmlc-core/include")
 include_directories("3rdparty/dlpack/include")
 
 
@@ -498,10 +497,6 @@ if(USE_CUDNN)
   endif()
 endif()
 
-if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/dmlc-core/cmake)
-  add_subdirectory("3rdparty/dmlc-core")
-endif()
-
 FILE(GLOB_RECURSE SOURCE "src/*.cc" "src/*.h" "include/*.h")
 FILE(GLOB_RECURSE CUDA "src/*.cu" "src/*.cuh")
 
@@ -894,7 +889,6 @@ install(TARGETS ${MXNET_INSTALL_TARGETS}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/dlpack/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/dmlc-core/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/mshadow/mshadow/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/mshadow)
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/mxnet/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/mxnet)
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/tvm/nnvm/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
